src/openms/source/ANALYSIS/ID/PeptideIndexing.cpp

```diff
@@ -368,9 +368,9 @@ PeptideIndexing::PeptideIndexing() :
 DefaultParamHandler("PeptideIndexing")
   {
 
-    defaults_.setValue("decoy_string", "_rev", "String that was appended (or prefixed - see 'prefix' flag below) to the accessions in the protein database to indicate decoy proteins.");
+    defaults_.setValue("decoy_string", "dec_", "String that was appended (or prefixed - see 'prefix' flag below) to the accessions in the protein database to indicate decoy proteins.");
 
-    defaults_.setValue("prefix", "false", "If set, protein accessions in the database contain 'decoy_string' as prefix.");
+    defaults_.setValue("prefix", "true", "If set, protein accessions in the database contain 'decoy_string' as prefix.");
```

Comment:
Ok - now that we change the PeptideIndexer interface, too. It is a good time to think about if it can be improved / made more stable for further additions. Any ideas?